src/gt_merger/args.py

```diff
@@ -28,20 +29,21 @@ def get_parser():
 
     parser.add_argument('--gtFile', type=str, required=True, help='Path to XLSX file including the Ground Truth data')
 
-    parser.add_argument('--dataDir', type=str, default='data',
-                        help='Folder including the sub-folder related to each GT Data files')
+    parser.add_argument('--dataDir', type=str, default=constants.DATA_DIR,
+                        help='Folder used to save output and log data from preprocess and merge')
```

Comment:
Maybe `dataDir` should be renamed to `outputDir`? Reading this description, it wasn't immediately obvious to me if this was an input folder that this script consumes data from or output folder where the result of this script is outputting data. The description in the README is better, but this should still contain a good explanation of what this parameter is.

Reply:
You are right. Initially dataDir was intended to holds the input files (assuming there were more than two). After adding the parameters for input files (oba and GT), then I decided to use it to group the output and log files generated by the script. 
I will rename it.